.gitignore

```diff
@@ -9,3 +9,4 @@ src/envo/envo-potential-redundant-module.owl
 src/envo/envo-edit-module-merge.owl
 src/envo/OntologyCheck-AfterLoad
 
+.DS_Store
```

Comment:
Looks good, this follows some online documentation?

Reply:
Yes: https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository

It looks like .DS_Store files are something Mac users should be aware of.

Reply:
Cool, thanks. Will merge.